### PR TITLE
Dlipeles feature/rewards banner

### PR DESCRIPTION
**Pull request by yachtcaptain23**
Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave-intl/publishers/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Tagged reviewers.
- [ ] Integrated piwik/matomo (for code that adds new buttons).
- [ ] Addressed or ignored all brakeman warnings

Test Plan:

Reviewer Checklist:

Tests
- [ ] Adequate test coverage exists to prevent regressions

Security:
- [ ] No raw SQL -- Always prefer ActiveRecord query helpers ([more info on StackOverflow](https://stackoverflow.com/questions/41410752/rails-5-sql-injection#41452695))
- [ ] XSS is mitigated -- Avoid `html_safe` and `raw`; escape untrusted content from users and 3rd party APIs ([Rails XSS guide](https://brakemanpro.com/2017/09/08/cross-site-scripting-in-rails) and [OWASP XSS Prevention Cheat Sheet](https://www.owasp.org/index.php/XSS_(Cross_Site_Scripting)_Prevention_Cheat_Sheet))

### CODE REVIEW

File: app/javascript/packs/banner_editor.jsx [hunk 1]
```diff
@@ -0,0 +1,646 @@
...
+        'Accept': 'text/html',
+        'X-Requested-With': 'XMLHttpRequest',
+        'X-CSRF-Token': document.head.querySelector("[name=csrf-token]").content
+        },
+        credentials: "same-origin",
+        }).then(function(response) {
+          return response.json();
+        })
+        .then(function(banner) {
+          console.log(banner);
```
Comment (yachtcaptain23) [inline app/javascript/packs/banner_editor.jsx]: Remove comments

File: app/javascript/packs/banner_editor.jsx [hunk 2]
```diff
@@ -0,0 +1,646 @@
...
+          <p onClick={ () => this.handleLinkDelete('Twitch') } style={{display:'inline', paddingLeft:'5px', cursor:'pointer', fontSize:'.85rem', color:'#7d7bdc'}}>(X)</p>
+        </div>
+      }
+    </div>
+  }
+
+  handleSave(event) {
+    let that = this
+    let id = document.getElementById("publisher_id").value;
+    console.log(id);
```
Comment (yachtcaptain23) [inline app/javascript/packs/banner_editor.jsx]: To remove

File: yarn.lock [hunk 3]
```diff
@@ -5,7 +5,6 @@
 "@rails/webpacker@3.5":
   version "3.5.5"
   resolved "https://registry.yarnpkg.com/@rails/webpacker/-/webpacker-3.5.5.tgz#8911c66bcefc8bc6b91270e92f0d39e3c2d43116"
-  integrity sha512-R9kglLb/R9mMqf2BW91XJHYc0q00qAO67vfrbV52ifTSywPOixEka/7oU3cb6671u1ajd74kuGAsCSnBtdVHJg==
```
Comment (nvonpentz) [inline yarn.lock]: This PR removes all the integrity sha's from yarn.lock

Comment (yachtcaptain23) [reply yarn.lock]: I did a fresh `yarn install` after deleting `yarn.lock` and I didn't see this integrity `sha`

Comment (nvonpentz) [reply yarn.lock]: @yachtcaptain23 is your yarn up-to-date? I think we probably want them.  cc @evq who added to the codebase originally https://github.com/brave-intl/publishers/pull/1189#issuecomment-424879428

**Pull request merged**